### DISCUSSION TITLE
Install instructions for legacy and non-legacy macOS

### DISCUSSION THREAD

**Discussion by OldSubSailor** [Ortho4XP/Ortho4XP#31] [General]
Do we want to mention the install instructions that have been written on the X-Plane forums here, or just let them ride there?
I try to keep the non-legacy updated when I discover something new, and I sure would not mind someone from this group visiting it to make sure that I have not missed anything.

**Comment by artesim-zz** [comment]
Hi,

I have absolutely no experience, and no access to macOS : in my mind, you're the expert here :)

But sure, I think there's a lot of valuable information in the forum, and it would be good to have a /doc folder for things like that, in addition to the README.md

...which gives me another idea ;-)

**Comment by OldSubSailor** [comment]
Well, for a start and review the link is: 
https://forums.x-plane.org/index.php?/forums/topic/244471-guide-for-installing-software-on-big-sur/

The first two posts are really all that are needed.
This alone is a good example for avoiding the forum. Regardless of what one says the things just grow wildly.

**Reply by OldSubSailor** [reply]
I hope that we have another Mac person around. Don't forget that I am 75 and there is Covid lurking about, old age, could get me. I have fought the big C once and there were other things.
